includes/admin/payments/orders.php

```diff
@@ -684,6 +684,29 @@ function edd_order_details_overview( $order ) {
 		}
 	}
 
+	$has_tax  = 0;
+	$tax_rate = $order->get_tax_rate();
+	if ( edd_is_add_order_page() && edd_use_taxes() ) {
+		$has_tax = array(
+			'rate'    => 0,
+			'country' => '',
+			'region'  => '',
+		);
```

Comment:
Code style suggestion: maybe define the fallback outside of the condition so the same value can be used in multiple places.